src/user/userRoutes.js

```diff
@@ -1,8 +1,12 @@
 const { Router } = require("express");
-const { addUser } = require("./userController");
+const { addUser, deleteUser } = require("./userController");
 const { hashPassword, checkPassword, checkEmail } = require("../middleware");
 const userRouter = Router();
 
-userRouter.post("/user", addUser);
+userRouter.post("/user", checkEmail, hashPassword, addUser);
+userRouter.delete("/deleteUser", deleteUser);
+userRouter.get("/user", listUsers);
+userRouter.put("/user", updateUsers);
```

Comment:
again, we'll only be able to update one user not multiple, so the name can be changed to singular (User)